oidc-ui/src/components/SignInOptions.js

```diff
@@ -57,24 +55,42 @@ export default function SignInOptions({
       )}
 
       {status.state === states.LOADED && singinOptions && (
-        <div className="divide-y-2">
-          {singinOptions.map((option, idx) => (
-            <div key={idx}>
-              <button
-                className="text-gray-500 font-semibold text-base"  id={`login_with_${option.label.toLowerCase()}`}
+        <div className="grid grid-rows-7 w-full flex rounded">
+          {singinOptions
+            .slice(0, showMoreOptions ? 4 : undefined)
+            .map((option, idx) => (
+              <div
+                key={idx}
+                className="w-full flex py-2 px-1 my-1 cursor-pointer"
+                style={boxStyle}
+                id={option.id}
                 onClick={(e) => handleSignInOptionClick(option.value)}
               >
-                <div className="flex items-center">
-                  <img className="w-7" src={option.icon} />
-                  <span className="ml-4 mr-4 mb-4 mt-4 ltr:text-left rtl:text-right">
-                    {t("login_with", {
-                      option: t(option.label),
-                    })}
-                  </span>
+                <img
+                  className="mx-2 h-6 w-6"
+                  src={option.icon}
+                  alt={option.id}
+                />
+                <div className="font-medium truncate ltr:text-left rtl:text-right ltr:ml-1.5 rtl:mr-1.5">
+                  {t("login_with", {
+                    option: t(option.label, option.label),
+                  })}
                 </div>
-              </button>
-            </div>
-          ))}
+              </div>
+            ))}
+        </div>
+      )}
+
+      {showMoreOptions && (
+        <div
+          className="text-center font-medium text-[#0953FA]"
+          onClick={() => setShowMoreOptions(false)}
+        >
+          {t("more_ways_to_sign_in")}
+          {/* <span className="mx-2 text-[#0953FA]">{"\u2304"}</span>
```

Comment:
can these comments be removed

Reply:
resolve in this [f23c62b](https://github.com/mosip/esignet/pull/367/commits/f23c62bca823f172be40fc5ad7a41974c9ebf45b)